client/main.lua

```diff
@@ -10,25 +10,31 @@ AddStateBagChangeHandler('qbx_fireworks:initiate', nil, function(bagName, _, val
     end
 end)
 
-local function startFirework(asset)
+local function startFirework(asset, entityCoords)
     local time
     local coords = GetEntityCoords(cache.ped)
 
     time = sharedConfig.detonationTime
 
     lib.requestNamedPtfxAsset(asset, 5000)
+    lib.requestNamedPtfxAsset('core', 5000)
```

Comment:
Do we need to release these particle effects after we're done with them? https://docs.fivem.net/natives/?_0x5F61EBBE1A00F96D

Reply:
Used cl2 and muted either one and it can be seen and heard. Also added the native to release both particle assets after use